Multisig/UI/Settings/OwnerKeyManagement/OwnerKeyDetailsViewController/OwnerKeyDetailsViewController.swift

```diff
@@ -109,7 +109,8 @@ class OwnerKeyDetailsViewController: UITableViewController, WebConnectionObserve
         tableView.registerCell(HelpLinkTableViewCell.self)
         tableView.registerHeaderFooterView(BasicHeaderView.self)
 
-        for notification in [Notification.Name.ownerKeyUpdated] {
+        for notification in [Notification.Name.ownerKeyUpdated,
```

Comment:
Notification.Name.ownerKeyUpdated could be also shortened to just .ownerKeyUpdated

Reply:
Unfortunately not.
<img width="741" alt="image" src="https://user-images.githubusercontent.com/246473/179704425-9b901d82-8e13-44e3-8e5f-2fddb04b436c.png">